frontend/src/lib/components/LemonSelect.tsx

```diff
@@ -7,7 +7,7 @@ import './LemonSelect.scss'
 import clsx from 'clsx'
 
 export interface LemonSelectOption<T> {
-    value: T
+    value?: T
```

Comment:
This made #11957 possible (see fix: #11958). It doesn't seem like `PlayerListRow`'s `options` are used anywhere yet – what should that menu look like? Maybe a `LemonMenu` component would be a better fit for it?